recipe/meta.yaml

```diff
@@ -84,7 +84,7 @@ extra:
 # Author: Daniel W. Heck [aut, cre]
 # Maintainer: Daniel W. Heck <heck@uni-mannheim.de>
 # Description: Estimates the precision of transdimensional Markov chain Monte Carlo  (MCMC) output, which is often used for Bayesian analysis of models with different  dimensionality (e.g., model selection). Transdimensional MCMC (e.g., reversible  jump MCMC) relies on sampling a discrete model-indicator variable to estimate  the posterior model probabilities. If only few switches occur between the models,  precision may be low and assessment based on the assumption of independent  samples misleading. Based on the observed transition matrix of the indicator  variable, the method of Heck, Overstall, Gronau, & Wagenmakers (2018,  Statistics & Computing) <doi:10.1007/s11222-018-9828-0> draws posterior samples  of the stationary distribution to (a) assess the uncertainty in the estimated  posterior model probabilities and (b) estimate the effective sample size of  the MCMC output.
-# License: GPL-3
+# License: GPL-3.0-only
```

Comment:
Don't change this here... that is the the original untouched r/cran meta data.. change this instead in line 59.